pypeit/spectrographs/gemini_flamingos.py

```diff
@@ -0,0 +1,340 @@
+"""
+Module for Gemini flamingos
+"""
+import numpy as np
+
+from pypeit import msgs
+from pypeit import telescopes
+from pypeit.core import framematch
+from pypeit.par import pypeitpar
+from pypeit.spectrographs import spectrograph
+
+
+class GeminiFLAMINGOSSpectrograph(spectrograph.Spectrograph):
+    """
+    Child to handle Gemini/Flamingos specific code
+
+    .. note::
+        For FIRE Echelle, we usually use high gain and SUTR read mode.
```

Comment:
FIRE?

Here and throughout, eliminate mention of Magellan/FIRE

Reply:
Removed.